### PR TITLE
Theme refactor

### PR DESCRIPTION
**Pull request by ferndot**
Introduces a new pattern that simplifies both the naming and usage of themes.

### CODE REVIEW

File: static/scss/components/Modal.scss [hunk 1]
```diff
@@ -1,136 +1,124 @@
...
+@import '../themes';
+
+@include themify() {
+  .modal-cover {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background: rgba(0, 0, 0, .6);
```
Comment (HR) [inline static/scss/components/Modal.scss]: This needs to be themified

File: static/scss/components/Modal.scss [hunk 2]
```diff
@@ -1,136 +1,124 @@
...
+    display: none;
+  }
+
+  .modal-cover.modal-cover-active {
+    display: block;
+    z-index: 1024;
+  }
+
+  .modal-container {
+    --background: themed('backgroundColor');
```
Comment (HR) [inline static/scss/components/Modal.scss]: No need for the variable now just use the theme bg color instead

**Review by HR — CHANGES_REQUESTED** [review]
Good work 👌
Just two minor things to polish it

File: static/scss/components/Modal.scss (hunk 1 above)
Comment (ferndot) [reply static/scss/components/Modal.scss]: Fixed!

File: static/scss/components/Modal.scss (hunk 2 above)
Comment (ferndot) [reply static/scss/components/Modal.scss]: Fixed!

**Review by HR — APPROVED** [review]

**Pull request merged**